cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -131,6 +131,8 @@ var (
 		Use:                   "set-config key1=val1 [key2=val2 ...]",
 		DisableFlagsInUseLine: true,
 		Short:                 "Add/update global config value in the NeoFS network",
+		Long: `Add/update global config value in the NeoFS network. 
+				If key is unknown, it will be added to the config only with --force flag.`,
```

Comment:
Add something like

Values for unknown keys are added exactly the way they're provided, no conversion is made ("123" will be stored as "313233" hexadecimal)

?